README.md

```diff
@@ -162,7 +162,7 @@ $ protoc googleapis/google/cloud/vision/v1/*.proto \
     --ruby_gapic_opt=configuration=vision.yml --ruby_gapic_out=dest
 ```
 
-Note that generated clients depend on the `gapic-common` 2.0 gem, which is not
+Note that generated clients depend on the `gapic-common` 0.2 gem, which is not
```

Comment:
Can you delete this paragraph, as it is no longer necessary? (The `gapic-common` gem is released.)